gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannel.java

```diff
@@ -488,6 +481,10 @@ public void beforeStart(
           ClientCallStreamObserver<WriteObjectRequest> clientCallStreamObserver) {
         clientCallStreamObserver.setOnReadyHandler(ready::countDown);
       }
+
+      public boolean isComplete() {
+        return done.getCount() != 0 ? false : true;
```

Comment:
Simplify to just `return done.getCount() == 0`

Reply:
Done.